docs/data/system/experimental-api/css-theme-variables/CreateCssVarsProvider.js

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
```

Comment:
Please duplicate this file for typescript with the same name `CreateCssVarsProvider.tsx` (next to this file). The infra will pick it up and the demo will show typescript first.

For example, Material UI button demo:

<img width="362" alt="image" src="https://github.com/mui/material-ui/assets/18292247/06e51d4e-12dd-4ed7-961e-3969a3843c1a">